style.css

```diff
@@ -0,0 +1,386 @@
+* {
+	box-sizing: border-box;
+	margin: 0;
+	font-size: 10px;
+	font-family: Ubuntu, Sans-serif;
+	color: white;
+	font-weight: bold;
+}
+br {
+	display:none;
+}
+body {
+	background-color: #C06C84 ;
+	width: 100%;
+	font-family: "Idim";
+}
+nav {
+	width: 100%;
+	display: flex;
+	justify-content: space-between;
+	height: 50px;
+	position: fixed;
+	padding: 15px;
+	background-color: #C06C84;
+}
+
+footer {
+	text-align: center;
+	background-color: #355C7D;
+	padding-bottom: 30px;
+}
+
+a:visited {
+	color: inherit;
+}
+
+
+
+.nav-left-container {
+	display: flex;
+	justify-content: flex-start;
+	gap: 30px;
+	color: white;
+}
+
+.nav-left-container a {
+	text-decoration: none;
+	font-size: 2rem;
+	color: white;
+}
+
+nav > a {
+	font-size: 2rem;
+	text-decoration:none;
+}
+
+.intro {
+	display: flex;
+	justify-content: center;
+	flex-wrap: wrap-reverse;
+	padding: 15px;
+	align-items: center;
+	max-width: 1000px;
+	margin: 0 auto;
+	height: 100%;
```

Comment:
i found out why this works:
![image](https://user-images.githubusercontent.com/25844355/213403103-e01765c5-28e9-48f0-b2d9-7c2e83d9ae6a.png)
here's the article (no need to read it): https://www.thoughtco.com/set-height-html-element-100-percent-3467075

Reply:
I see! Thanks for the link